pkg/network/tracer/connection/util/conn_tracer.go

```diff
@@ -93,8 +93,6 @@ func SetupHandler(eventHandler ebpf.EventHandler, mgr *ebpf.Manager, cfg *config
 				RecordGetter:     handler.RecordGetter,
 				RecordHandler:    handler.RecordHandler,
 				TelemetryEnabled: cfg.InternalTelemetryEnabled,
-				// RingBufferSize is not used yet by the manager, we use a map editor to set it in the tracer
```

Comment:
I think the perfbuffer variant of this type actually does support the size field. Is it very difficult to add it to the ringbuffer so we have consistency?

Reply:
With perf buffer: max entries must be <= nCPU, and does not reflect size of buffers
With ring buffer: max entries is the size of the buffer. Having two fields mean the same thing would be confusing.

Reply:
We wouldn't know whether to use max entries or RingBufferSize as the true size.

Reply:
@akarpz does that make sense?

Reply:
sorry, yes. approving now